Sources/App/Gfs/GfsVariable.swift

```diff
@@ -158,6 +159,7 @@ enum GfsSurfaceVariable: String, CaseIterable, GenericVariable, GenericVariableM
         case .categorical_freezing_rain: return 1
         case .convective_inhibition: return 1
         case .total_column_integrated_water_vapour: return 10
+        case .mass_density_8m: return 1
```

Comment:
Could you set the scale factor to `0.1`? With values up to 10.000 µg/m³  it could get close to an internal 16-bit limit for compressing data which might lead to overflows.